package/base-files/files/sbin/sysupgrade

```diff
@@ -161,35 +175,37 @@ add_overlayfiles() {
 
 		# backup files from /etc/sysupgrade.conf and /lib/upgrade/keep.d, but
 		# ignore those aready controlled by opkg conffiles
-		list_static_conffiles | sort -u |
-			grep -h -v -x -F -f $conffiles > "$keepfiles"
+		list_static_conffiles | sort -u \
+			| grep -h -v -x -F -f $conffiles > "$keepfiles"
 
 		# backup conffiles, but only those changed if '-u'
-		[ $SKIP_UNCHANGED = 1 ] &&
-			list_changed_conffiles | sort -u > "$conffiles"
+		[ $SKIP_UNCHANGED = 1 ] \
+			&& list_changed_conffiles | sort -u > "$conffiles"
```

Comment:
Single line.

Reply:
I don't want to exceed 88 chars per line.

Reply:
Just as a clarification I'm not particularly interested in the number, it's just what the Python linter `black` uses as default.